vendor.conf

```diff
@@ -77,7 +77,7 @@ golang.org/x/sys master
 golang.org/x/text f72d8390a633d5dfb0cc84043294db9f6c935756
 golang.org/x/time f51c12702a4d776e4c1fa9b0fabab841babae631
 golang.org/x/sync master
-google.golang.org/grpc v1.0.4 https://github.com/grpc/grpc-go<
+google.golang.org/grpc v1.0.4 https://github.com/grpc/grpc-go
```

Comment:
The tag didn't change, but some files did. Is this correct? It looks fishy, but maybe that's a release branch.

Reply:
grpc was not revendored. I have only included a fix for a typo as part of this PR.

Reply:
I should grab another coffee, I didn't notice this. Thanks.